src/react-components/link-root.js

```diff
@@ -208,15 +211,19 @@ class LinkRoot extends Component {
                   {d}
                 </button>
               ))}
-              <button
-                className={classNames(styles.keypadButton, styles.keypadToggleMode)}
-                onTouchStart={() => this.toggleMode()}
-                onClick={() => {
-                  if (!hasTouchEvents) this.toggleMode();
-                }}
-              >
-                {this.state.isAlphaMode ? "123" : "ABC"}
-              </button>
+              {this.props.showHeadsetLinkOption ? (
+                <button
+                  className={classNames(styles.keypadButton, styles.keypadToggleMode)}
+                  onTouchStart={() => this.toggleMode()}
+                  onClick={() => {
+                    if (!hasTouchEvents) this.toggleMode();
+                  }}
+                >
+                  {this.state.isAlphaMode ? "123" : "ABC"}
+                </button>
+              ) : (
+                <div />
+              )}
```

Comment:
Not sure if you noticed, but the "Linking a Headset?" option in the footer is currently styled to disappear on short mobile screens. Even if it didn't just disappear, you wouldn't be able to scroll to it since we disable scrolling on this page.

Reply:
yeah that option should only be visible inside of an HMD